docs/legacy_css.md

```diff
@@ -10,24 +10,7 @@ Please note that this documentation is now deprecated and will be removed with t
 
 Tech-debt creep in CSS is usually an symptom of a lack of confidence in changing or removing existing CSS. By structuring CSS in the following way, we can help reduce this risk by clearly communicating the scope of any given CSS.
 
-The stylesheets are split between frontend and admin. The frontend ones are in a much better state and should be the way the admin ones move in the future.
```

Comment:
Do we have any context on this line? Basically just worried that we're losing some knowledge here about the state of our JS files and planned improvements to them

Reply:
My hope was that all the admin CSS would be replaced in the migration to the design system (and this line has remained [untouched for 10 years](https://github.com/alphagov/whitehall/commit/13fd302491145bb49b9a82290f4e179ca38b05b5)).